packages/run-protocol/test/test-feeDistributor.js

```diff
@@ -8,8 +8,8 @@ import { setup } from '@agoric/zoe/test/unitTests/setupBasicMints.js';
 
 import { makePromiseKit } from '@endo/promise-kit';
 import { assertPayoutAmount } from '@agoric/zoe/test/zoeTestHelpers.js';
-import { E } from '@endo/far';
-import { buildDistributor } from '../src/distributeFees.js';
+import { E, Far } from '@endo/far';
+import { makeFeeDistributor } from '../src/feeDistributor.js';
```

Comment:
I like that you're testing the object. is anything testing the contract?

Reply:
> I like that you're testing the object. is anything testing the contract?

Yes, the `run-protocol/test/test-gov-collaterals.js` is testing it.